### ISSUE TITLE
Failed to activate the dart-tools package

### ISSUE THREAD

**Issue by TheSeamau5** [radicaled/dart-tools#22]
[Enter steps to reproduce below:]

1. Restarted and updated Atom
2. Red square appeared as soon as Atom opened

**Atom Version**: 0.187.0
**System**: Mac OS X 10.10.2
**Thrown From**: [dart-tools](https://github.com/radicaled/dart-tools) package, v0.8.8

### Stack Trace

Failed to activate the dart-tools package

```
At Cannot read property 'getFile' of undefined

TypeError: Cannot read property 'getFile' of undefined
    at Function.module.exports.Utils.isDartProject (/Users/hassanhayat/.atom/packages/dart-tools/lib/utils.coffee:64:49)
    at Object.module.exports.activate (/Users/hassanhayat/.atom/packages/dart-tools/lib/dart-tools.coffee:17:18)
    at Package.module.exports.Package.activateNow (/Applications/Atom.app/Contents/Resources/app/src/package.js:240:19)
    at /Applications/Atom.app/Contents/Resources/app/src/package.js:221:30
    at Package.module.exports.Package.measure (/Applications/Atom.app/Contents/Resources/app/src/package.js:167:15)
    at Package.module.exports.Package.activate (/Applications/Atom.app/Contents/Resources/app/src/package.js:214:14)
    at PackageManager.module.exports.PackageManager.activatePackage (/Applications/Atom.app/Contents/Resources/app/src/package-manager.js:465:21)
    at /Applications/Atom.app/Contents/Resources/app/src/package-manager.js:446:29
    at Config.module.exports.Config.transact (/Applications/Atom.app/Contents/Resources/app/src/config.js:350:16)
    at PackageManager.module.exports.PackageManager.activatePackages (/Applications/Atom.app/Contents/Resources/app/src/package-manager.js:440:19)
```

### Commands

```
```

### Config

```json
{
  "core": {
    "disabledPackages": [
      "linter-gjslint",
      "linter",
      "linter-dartanalyzer"
    ],
    "themes": [
      "unity-ui",
      "xcode-syntax"
    ]
  }
}
```

### Installed Packages

```coffee
# User
autoclose-html, v0.15.0
autocomplete-plus, v2.4.2
color-picker, v1.4.4
dart-tools, v0.8.8
file-icons, v1.5.1
linter-csslint, v0.0.11
linter-tslint, v0.2.1
minimap, v4.6.0
unity-ui, v2.0.5
xcode-syntax, v0.1.0

# Dev
No dev packages
```

**Comment by radicaled** [comment]
I believed this is fixed in 0.8.9 -- let me know if it's not.

**Comment by TheSeamau5** [comment]
ok
